src/main/kotlin/com/workos/organizations/OrganizationsApi.kt

```diff
@@ -14,12 +14,39 @@ import com.workos.organizations.models.OrganizationList
  * The OrganizationsApi provides convenience methods for working with WorkOS Organizations.
  */
 class OrganizationsApi(private val workos: WorkOS) {
+  /**
+   * Options when setting the domains for an organization.
+   *
+   * @param domain The domain of the organization.
+   * @param state The verificaction state of the domain.
+   */
+  class OrganizationDomainDataOptions @JvmOverloads constructor(
+    val domain: String,
+
+    val state: OrganizationDomainDataState
+  )
+
+  /**
+   * The verification state of the domain
+   *
+   * @property PENDING The domain is pending verification.
+   * @property VERIFIED The domain has been verified. Pass this if you've already verified the domain through other methods.
+   */
+  enum class OrganizationDomainDataState {
+    @JsonProperty("verified")
+    VERIFIED,
```

Comment:
Another nit, no other enum uses all uppercase to define possible values in an enum, they all use first letter in the word as uppercase, rest as lowercase.

Reply:
Ah, I hadn't seen the others. Will keep it consistent.